### ISSUE TITLE
Rename "DAML SDK" to "SDK", "DAML Connect" or "DAML Connect SDK" as appropriate.

### ISSUE THREAD

**Issue by bame-da** [digital-asset/daml#7612]
We would like to phase out the term "DAML SDK" and restrict "SDK" to refer to developer tools only - ie not libraries like language bindings, or runtime components like the JSON API. That overall umbrella will be called "DAML Connect".

Please have a look for occurrences of "DAML SDK" and "SDK" in our artifacts (other than docs) and change them as appropriate: 

- If it's repo-internal only, ignore.
- If the cost of change is high, or if it would result in backwards incompatible API changes, please just flag the places where "DAML SDK" or inaccurate "SDK" nomenclature will remain in this ticket.
- Otherwise, replace the term with "SDK", "DAML Connect" or "DAML Connect SDK" as appropriate.

**Comment by cocreature** [comment]
Query
```
rg --sort path -l -g '!docs' -g '!daml.yaml*' -g '!*test*' -i sdk
```
On 51e53ce00673aa39b924cfb1d43ba23768dadeda

Results:
- [] BUILD
- [] NOTICES
- [] README.md
- [] WORKSPACE
- [] azure-cron.yml
- [] azure-pipelines.yml
- [] bazel_tools/build_environment.bzl
- [] bazel_tools/haskell-strict-source-names.patch
- [] bazel_tools/nixpkgs-hermetic-cc-toolchain.patch
- [] bazel_tools/pom_template.xml
- [] build-scripts/build_hs_lf_tooling.sh
- [] ci/build-unix.yml
- [] ci/build-windows.yml
- [] ci/compatibility-windows.yml
- [] ci/compatibility.yml
- [] ci/cron/src/Main.hs
- [] compatibility/BUILD
- [] compatibility/README.md
- [] compatibility/WORKSPACE
- [] compatibility/assistant/BUILD
- [] compatibility/assistant/src/DA/Test/PlatformVersion.hs
- [] compatibility/bazel_tools/daml.cc.tpl
- [] compatibility/bazel_tools/daml_ledger/Main.hs
- [] compatibility/bazel_tools/daml_script/daml_script.bzl
- [] compatibility/bazel_tools/daml_sdk.bzl
- [] compatibility/bazel_tools/daml_trigger/daml_trigger.bzl
- [] compatibility/bazel_tools/data_dependencies/data_dependencies.bzl
- [] compatibility/bazel_tools/sandbox-with-postgres.sh
- [] compatibility/build-release-artifacts-windows.ps1
- [] compatibility/build-release-artifacts.sh
- [] compatibility/build-ts-libs.sh
- [] compatibility/deps.bzl
- [] compatibility/sandbox-migration/BUILD
- [] compatibility/sandbox-migration/runner/Migration/Divulgence.hs
- [] compatibility/sandbox-migration/runner/Migration/KeyTransfer.hs
- [] compatibility/sandbox-migration/runner/Migration/ProposeAccept.hs
- [] compatibility/sandbox-migration/runner/Migration/Runner.hs
- [] compatibility/sandbox-migration/runner/Migration/Types.hs
- [] compatibility/sandbox-migration/util.bzl
- [] compatibility/versions/UpdateVersions.hs
- [] compatibility/versions.bzl
- [] compiler/daml-extension/README.md
- [] compiler/daml-extension/src/extension.ts
- [] compiler/daml-lf-reader/src/DA/Daml/LF/Reader.hs
- [] compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/NameCollision.hs
- [] compiler/daml-licenses/BUILD.bazel
- [] compiler/daml-licenses/PACKAGE
- [] compiler/daml-licenses/licenses/extract.py
- [] compiler/daml-licenses/licenses/gogol-core.cabal
- [] compiler/daml-licenses/licenses/gogol-logging.cabal
- [] compiler/daml-licenses/licenses/gogol.cabal
- [] compiler/daml-licenses/licenses/licensing.md.tmpl
- [] compiler/daml-licenses/source/index.rst
- [] compiler/daml-licenses/source/licensing/da-theme-licenses.rst
- [] compiler/daml-licenses/source/licensing/docs-licenses.rst
- [] compiler/damlc/BUILD.bazel
- [] compiler/damlc/README.md
- [] compiler/damlc/base-rst-index-template.rst
- [] compiler/damlc/daml-compiler/BUILD.bazel
- [] compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Dar.hs
- [] compiler/damlc/daml-compiler/src/DA/Daml/Compiler/DataDependencies.hs
- [] compiler/damlc/daml-compiler/src/DA/Daml/Compiler/DecodeDar.hs
- [] compiler/damlc/daml-compiler/src/DA/Daml/Compiler/ExtractDar.hs
- [] compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Fetch.hs
- [] compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Output.hs
- [] compiler/damlc/daml-ide-core/BUILD.bazel
- [] compiler/damlc/daml-ide-core/src/Development/IDE/Core/Rules/Daml.hs
- [] compiler/damlc/daml-lf-conversion/BUILD.bazel
- [] compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion/UtilGHC.hs
- [] compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs
- [] compiler/damlc/daml-opts/BUILD.bazel
- [] compiler/damlc/daml-opts/daml-opts/DA/Daml/Options.hs
- [] compiler/damlc/daml-package-config/BUILD.bazel
- [] compiler/damlc/daml-package-config/src/DA/Daml/Package/Config.hs
- [] compiler/damlc/daml-preprocessor/BUILD.bazel
- [] compiler/damlc/daml-preprocessor/src/DA/Daml/Preprocessor/EnumType.hs
- [] compiler/damlc/daml-preprocessor/src/DA/Daml/Preprocessor/Generics.hs
- [] compiler/damlc/daml-preprocessor/src/DA/Daml/Preprocessor.hs
- [] compiler/damlc/daml-rule-types/BUILD.bazel
- [] compiler/damlc/daml-stdlib-src/DA/Generics.daml
- [] compiler/damlc/lib/DA/Cli/Damlc/BuildInfo.hs
- [] compiler/damlc/lib/DA/Cli/Damlc/Packaging.hs
- [] compiler/damlc/lib/DA/Cli/Damlc.hs
- [] compiler/damlc/pkg-db/util.bzl
- [] daml-assistant/daml-helper/BUILD.bazel
- [] daml-assistant/daml-helper/src/DA/Daml/Helper/Codegen.hs
- [] daml-assistant/daml-helper/src/DA/Daml/Helper/Init.hs
- [] daml-assistant/daml-helper/src/DA/Daml/Helper/Ledger.hs
- [] daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs
- [] daml-assistant/daml-helper/src/DA/Daml/Helper/New.hs
- [] daml-assistant/daml-helper/src/DA/Daml/Helper/Start.hs
- [] daml-assistant/daml-helper/src/DA/Daml/Helper/Studio.hs
- [] daml-assistant/daml-helper/src/DA/Daml/Helper/Util.hs
- [] daml-assistant/daml-project-config/DA/Daml/Project/Config.hs
- [] daml-assistant/daml-project-config/DA/Daml/Project/Consts.hs
- [] daml-assistant/daml-project-config/DA/Daml/Project/Types.hs
- [] daml-assistant/daml-sdk/BUILD.bazel
- [] daml-assistant/daml-sdk/src/main/scala/com/digitalasset/daml/sdk/SdkMain.scala
- [] daml-assistant/exe/DA/Daml/Assistant.hs
- [] daml-assistant/get-daml.sh
- [] daml-assistant/scala-daml-project-config/BUILD.bazel
- [] daml-assistant/scala-daml-project-config/README.md
- [] daml-assistant/scala-daml-project-config/src/main/scala/com/digitalasset/assistant/config/ProjectConfig.scala
- [] daml-assistant/src/DA/Daml/Assistant/Cache.hs
- [] daml-assistant/src/DA/Daml/Assistant/Command.hs
- [] daml-assistant/src/DA/Daml/Assistant/Env.hs
- [] daml-assistant/src/DA/Daml/Assistant/Install/Github.hs
- [] daml-assistant/src/DA/Daml/Assistant/Install.hs
- [] daml-assistant/src/DA/Daml/Assistant/Types.hs
- [] daml-assistant/src/DA/Daml/Assistant/Version.hs
- [] daml-lf/interface/BUILD.bazel
- [] daml-lf/spec/daml-lf-1.rst
- [] daml-lf/spec/value.rst
- [] daml-lf/transaction/src/main/protobuf/com/daml/lf/value.proto
- [] daml-script/daml/BUILD.bazel
- [] daml-script/daml/Daml/Script.daml
- [] daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Runner.scala
- [] deps.bzl
- [] dev-env/bin/daml-sdk-head
- [] ghc-lib/working-on-ghc-lib.md
- [] infra/vsts_agent_windows.tf
- [] language-support/codegen-common/src/main/scala/com/digitalasset/daml/lf/codegen/conf/CodegenConfigReader.scala
- [] language-support/hs/bindings/examples/nim/demo.md
- [] language-support/java/codegen/BUILD.bazel
- [] language-support/java/codegen/README.md
- [] language-support/scala/codegen-sample-app/src/main/daml/MyMain.daml
- [] language-support/scala/examples/iou-no-codegen/build.sbt
- [] language-support/scala/examples/iou-no-codegen/project/Versions.scala
- [] language-support/scala/examples/quickstart-scala/README.md
- [] language-support/scala/examples/quickstart-scala/build.sbt
- [] language-support/scala/examples/quickstart-scala/project/Versions.scala
- [] language-support/ts/codegen/README.md
- [] language-support/ts/codegen/src/TsCodeGenMain.hs
- [] language-support/ts/daml-ledger/BUILD.bazel
- [] language-support/ts/daml-ledger/README.md
- [] language-support/ts/daml-ledger/package.json
- [] language-support/ts/daml-react/BUILD.bazel
- [] language-support/ts/daml-react/README.md
- [] language-support/ts/daml-react/package.json
- [] language-support/ts/daml-types/BUILD.bazel
- [] language-support/ts/daml-types/README.md
- [] language-support/ts/daml-types/package.json
- [] language-support/ts/typedoc.bzl
- [] ledger/participant-integration-api/src/main/scala/platform/apiserver/GrpcServer.scala
- [] libs-haskell/da-hs-base/BUILD.bazel
- [] libs-haskell/da-hs-base/da-hs-base.cabal
- [] libs-haskell/da-hs-base/src/DA/Service/Logger/Impl/GCP.hs
- [] navigator/README.md
- [] navigator/backend/src/main/scala/com/digitalasset/navigator/config/Config.scala
- [] navigator/frontend/yarn.lock
- [] nix/bazel.nix
- [] nix/certs/ca-bundle.pem
- [] nix/default.nix
- [] nix/overrides/ghc-8.6.5.nix
- [] nix/tools/bazel-cc-toolchain/default.nix
- [] nix/tools/nix-store-gcs-proxy/go.sum
- [] oss-compliance/check_oss_license.sh
- [] release/BUILD.bazel
- [] release/RELEASE.md
- [] release/sdk-config.yaml.tmpl
- [] release/src/Main.hs
- [] release/util.bzl
- [] release/windows-installer/BUILD.bazel
- [] release/windows-installer/src/WindowsInstaller.hs
- [] rules_daml/daml.bzl
- [] templates/README.txt
- [] triggers/daml/BUILD.bazel
- [] triggers/service/BUILD.bazel
- [] yarn.lock

Just tick off whatever you’ve checked. Don’t be too scared by the long list, most of it is trivial.

**Comment by cocreature** [comment]
Fixed in #7624 and #7615. Docs are still missing but this issue explicitly excludes docs.
